src/main/java/com/kryptnostic/rhizome/mapstores/cassandra/AbstractStructuredCassandraPartitionKeyValueStore.java

```diff
@@ -27,4 +32,15 @@ protected RegularStatement loadQuery(){
     protected RegularStatement deleteQuery(){
         return tableBuilder.buildDeleteByPartitionKeyQuery();
     }
+  
+    /**
+     * We assume that the (partition key)-value store has a unique value for each partition key. This has to be true for a (partition key)-value store to make sense.
+     */
+    @Override
+    public Map<K, V> loadAll( Collection<K> keys ) {
```

Comment:
Hazelcast samples suggest that returning null is okay... can you do a quick test and make see if this is the case? 

You can just use an <Integer,Integer> mapstore that returns randomly null values.